django_ibmi/operations.py

```diff
@@ -207,15 +178,24 @@ def date_interval_sql(self, timedelta):
             timedelta.days, timedelta.seconds, timedelta.microseconds), []
 
     # As casting is not required, so nothing is required to do in this function.
-    def datetime_cast_sql(self):
+    def datetime_cast_date_sql(self, field_name, tzname):
         return "%s"
 
     def deferrable_sql(self):
-        return ""
+        return "ON DELETE NO ACTION NOT ENFORCED"
+
+    def datetime_cast_time_sql(self, field_name, tzname):
+        return "%s"
 
-    # Function to return SQL from dropping foreign key.
-    def drop_foreignkey_sql(self):
-        return "DROP FOREIGN KEY"
+    def time_trunc_sql(self, lookup_type, field_name):
```

Comment:
Looks like this is missing `tzname` parameter: https://github.com/django/django/blob/76c0b32f826469320c59709d31e2f2126dd7c505/django/db/backends/postgresql/operations.py#L75